tests/unit/modules/dnac/fixtures/inventory_workflow_manager.json

```diff
@@ -0,0 +1,642 @@
+{
```

Comment:
This is NOT readable. Can't we write this json more readable?

```
{
  "playbook_add_device": [
    {
      "cli_transport": "ssh",
      "compute_device": false,
      "enable_password": "Cisco#123",
      "extended_discovery_info": null,
      "http_password": null,
      "http_port": null,
      "http_secure": null,
      "http_username": null,
      "ip_address_list": [
        "60.1.1.1",
        "50.1.1.1"
      ],
      "hostname_list": null,
      "mac_address_list": null,
      "netconf_port": "830",
      "password": "Cisco#123",
      "serial_number_list": null,
      "snmp_auth_passphrase": null,
      "snmp_auth_protocol": "SHA",
      "snmp_mode": null,
      "snmp_priv_passphrase": null,
      "snmp_priv_protocol": null,
      "snmp_ro_community": "test123",
      "snmp_rw_community": "test123",
      "snmp_retry": 3,
      "snmp_timeout": 5,
      "snmp_username": null,
      "snmp_version": "v2",
      "update_mgmt_ipaddresslist": null,
      "username": "cisco",
      "role": null,
      "device_resync": null,
      "reboot_device": null,
      "credential_update": null,
      "export_device_details_limit": 500,
      "resync_device_count": 200,
      "resync_max_timeout": 600,
      "force_sync": null,
      "clean_config": null,
      "add_user_defined_field": null,
      "update_interface_details": null,
      "export_device_list": null,
      "provision_wired_device": null
    }
  ],
..
...
```

Reply:
updated the code